app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_likes_counter
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
 end
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think It would be nice to   make the `update_post_counter` method privately as the current one is `public`. For example:
```ruby
  ...
  # app/models/post.rb
  ...
  after_save :update_likes_counter

  private

  def update_likes_counter
    author.increment!(:posts_counter)
  end
```